atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SyncSnapshotTransactionTest.java

```diff
@@ -193,43 +191,26 @@
 import org.jmock.Mockery;
 import org.jmock.lib.concurrent.DeterministicScheduler;
 import org.jmock.lib.concurrent.Synchroniser;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 import org.mockito.InOrder;
 import org.mockito.Mockito;
 
-/* TODO(boyoruk): Migrate to JUnit5 */
 @SuppressWarnings("checkstyle:all")
-@RunWith(Parameterized.class)
-public class SnapshotTransactionTest extends AtlasDbTestCase {
```

Comment:
See `AsyncSnapshotTransactionTest` for other parameter.

Reply:
There still seems to be a lot of unnecessry duplication between these classes, though. Can't we have an `AbstractSnapshotTransactionTest` that both inherit from with the common logic?